grafana/meta/collectd.yml

```diff
@@ -0,0 +1,10 @@
+{%- if pillar.grafana.server is defined %}
```

Comment:
I guess it's ok as `server.get('enabled', False)` will evaluate to `False`.

I've raised this because I don't think it's a common pattern in meta yml files.